### PR TITLE
chore: automatically import Grafana dashboards when installing gateway-addons-helm

### PR DESCRIPTION
**Pull request by shawnh2**
**What type of PR is this?**

**What this PR does / why we need it**:

When deploying `gateway-addons-helm` successfully, we will get dashboards automatically import:

image

**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

File: charts/gateway-addons-helm/values.yaml [hunk 1]
```diff
@@ -10,6 +11,38 @@ grafana:
...
+      apiVersion: 1
+      providers:
+        - name: 'envoy-gateway'
+          orgId: 1
+          folder: 'envoy-gateway'
+          type: file
+          disableDeletion: false
+          editable: true
+          options:
+            path: /var/lib/grafana/dashboards/envoy-gateway
```
Comment (zhaohuabing) [inline charts/gateway-addons-helm/values.yaml:25]: I guess the dashboards in the followings configuration will be downloaded and placed in this directory by grafana?

File: charts/gateway-addons-helm/values.yaml [hunk 2]
```diff
@@ -10,6 +11,38 @@ grafana:
...
+  dashboards:
+    envoy-gateway:
+      prometheus-stats:
+        gnetId: 1
+        revision: 1
+        datasource: Prometheus
+      # Scrape and load dashboards of Envoy Gateway
+      envoy-global-dashboard:
+        url: https://raw.githubusercontent.com/envoyproxy/gateway/main/charts/gateway-addons-helm/dashboards/envoy-global.json
```
Comment (arkodg) [inline charts/gateway-addons-helm/values.yaml:34]: is this the best way to do it ? or should the dashboard we part of this chart ?

File: charts/gateway-addons-helm/values.yaml (hunk 1 above)
Comment (zhaohuabing) [reply charts/gateway-addons-helm/values.yaml:25]: Oh, it's not.

File: charts/gateway-addons-helm/values.yaml (hunk 2 above)
Comment (arkodg) [reply charts/gateway-addons-helm/values.yaml:34]: the drawback of URL is, its not versioned

Comment (shawnh2) [reply charts/gateway-addons-helm/values.yaml:34]: got it, prefer loading dashboards from local disk

Comment (shawnh2) [reply charts/gateway-addons-helm/values.yaml]: change it to load dashboards from configmap

**Review by arkodg — APPROVED** [review]
LGTM thanks !

File: charts/gateway-addons-helm/values.yaml [hunk 3]
```diff
@@ -10,6 +11,21 @@ grafana:
   adminPassword: admin
   service:
     type: LoadBalancer
+  dashboardProviders:
```
[the remark this answers is not in the document]
Comment (shawnh2) [reply charts/gateway-addons-helm/values.yaml:14]: yes, grafana will need to hold the imported dashboards from configmap according to the provider path.

**Review by zirain — APPROVED** [review]

**Pull request merged**